src/rtems/timer/bench_riscv_machine_timer.c

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
```

Comment:
Is this date right? Anyway, is this the very same one file for zephyr?

Reply:
Good catch. Should be 2022.